modules/learn/pages/clusters-and-availability/xdcr-viewing-conflict-logs.adoc

```diff
@@ -0,0 +1,348 @@
+
+[source,sqlpp]
+----
+SELECT count(*) AS num_conflicts
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META().id LIKE "crd%" and b.timestamp between "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+*Query example 3:* Number of different `docIds` that had conflicts between two time periods.
+
+[source,sqlpp]
+----
+SELECT count (distinct b.docId) as num_different_docIds
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META().id LIKE "crd%" and b.timestamp between "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+*Query example 4:* List all `docIds` and the number of times each `docId` had a logged conflict between two time periods.
+
+[source,sqlpp]
+----
+SELECT b.docId AS DOCID, COUNT(1) AS num_times_seen
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META(b).id LIKE "crd%" AND b.timestamp BETWEEN "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+GROUP BY b.docId
+ORDER BY num_times_seen DESC LIMIT 20
+----
+
+The query result is as follows:
+
+----
+[
+  {
+    "DOCID": "4d9d18e7-5fa2-4eae-b32b-60d3d369da94",
+    "num_times_seen": 2   
+  },
+  {
+    "DOCID": "ae03ecde-203c-4a3b-a6d0-36b60f627c8a",
+    "num_times_seen": 1   
+  }
+]
+----
+
+*Query example 5:* List all `docIds` and fields from the `src` and `tgt` documents that were in conflict, to view only the `docId` that had conflict with select fields from the `src` and `tgt` documents.
+
+[source,sqlpp]
+----
+SELECT b.docId, b.srcDoc.id AS srcDocId, b.tgtDoc.id AS tgtDocId,
+(SELECT b2.country, b2.note
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b2 USE KEYS b.srcDoc.id)[0] AS src_doc_fields,
+(SELECT b3.country, b3.note
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b3 USE KEYS b.tgtDoc.id)[0] AS tgt_doc_fields
+FROM `conflictlogs`.`bucket1`.`app1logs` AS b
+WHERE META(b).id LIKE "crd%" AND b.timestamp BETWEEN "2025-06-04T18:15:16.363Z" AND "2025-06-09T07:26:40.326Z"
+----
+
+The query result is as follows:
+----
+[
+   {
+    "docId": "ae03ecde-203c-4a3b-a6d0-36b60f627c8a",
+    "srcDocId": "src_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+    "tgtDocId": "tgt_1749454000005461807_e2f416ed317beb4f16a3587656b3d3450e1bec893cabebd3b9bb857d63824da4",
+    "src_doc_fields": {
+        "country": "UK",
+        “note”: “need review”
+      },
+    "tgt_doc_fields": {
+        "country": "CA",
+        “note”: “ok”
+      } 
+   }
+]
+----
+
+NOTE: As the conflict log documents won’t be replicated, if you need to replicate them, copy them into a different document.
+
+[#using-eventing-for-conflict-logs]
+== Using Eventing for Conflict Logs
+
+The conflict logs and conflict documents are not replicated as XDCR’s special attribute (system xattrs called `_xdcr_conflict`) prevents replication:
+
+----
+"xattrs": {
+    "_xdcr_conflict": true
+    }
+----
+
+To force replicate the conflict logs and conflict documents, do the following:
```

Comment:
Please remove lines 143 to 151.   We don't need to detail this info with such prominence -- I recall that it's documented already in the Conflict Record Document section in the Conflict Logging page.